cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/ExistingDataprocProvisioner.java

```diff
@@ -145,6 +145,7 @@ public Cluster getClusterDetail(ProvisionerContext context, Cluster cluster) {
 
   @Override
   public PollingStrategy getPollingStrategy(ProvisionerContext context, Cluster cluster) {
-    return PollingStrategies.fixedInterval(0, TimeUnit.SECONDS);
+    DataprocConf conf = DataprocConf.create(createContextProperties(context));
+    return PollingStrategies.fixedInterval(conf.getJobStatusPollInterval(), TimeUnit.SECONDS);
```

Comment:
do you know if we end up using this interval for creation parts? If so, it would be better to check the cluster status like the ephemeral one does, so we don't add a couple seconds to the start up time.

Reply:
Done.